packages/dapp/src/tokenSupport/tokenSchema.json

```diff
@@ -2,43 +2,309 @@
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/BAT.png?alt=media"
+    },
+    {
+      "tokenContract": "0xDe30da39c46104798bB5aA3fe8B9e0e1F348163F",
+      "symbol": "GTC",
+      "decimals": 18,
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/GTC.png?alt=media"
+    },
+    {
+      "tokenContract": "0x6810e776880C02933D47DB1b9fc05908e5386b96",
+      "symbol": "GNO",
+      "decimals": 18,
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/GNO.png?alt=media"
+    },
+    {
+      "tokenContract": "0xf2051511B9b121394FA75B8F7d4E7424337af687",
+      "symbol": "HAUS",
+      "decimals": 18,
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/HAUS.png?alt=media"
+    },
+    {
+      "tokenContract": "0xfb5453340c03db5ade474b27e68b6a9c6b2823eb",
+      "symbol": "ROBOT",
+      "decimals": 18,
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/ROBOT.png?alt=media"
     }
   ],
   "4": [
     {
       "tokenContract": "0xc778417E063141139Fce010982780140Aa0cD5Ab",
-      "decimals": "18",
       "symbol": "WETH",
-      "image": "https://github.com/raid-guild/smart-invoice/blob/tokenImages/weth.png?raw=true"
+      "decimals": 18,
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/WETH.png?alt=media"
     },
     {
       "tokenContract": "0x3af6b2f907f0c55f279e0ed65751984e6cdc4a42",
-      "decimals": "18",
       "symbol": "DAI",
-      "image": "https://github.com/raid-guild/smart-invoice/blob/tokenImages/dai.png?raw=true"
+      "decimals": 18,
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/DAI.png?alt=media"
     },
     {
       "tokenContract": "0x982e00B16c313E979C0947b85230907Fce45d50e",
-      "decimals": "18",
       "symbol": "TEST",
-      "image": "https://github.com/raid-guild/smart-invoice/blob/tokenImages/test.png?raw=true"
+      "decimals": 18,
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/TEST.png?alt=media"
+    }
+  ],
+  "5": [
+    {
+      "tokenContract": "0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6",
+      "symbol": "WETH",
+      "decimals": 18,
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/WETH.png?alt=media"
+    },
+    {
+      "tokenContract": "0x73967c6a0904aA032C103b4104747E88c566B1A2",
+      "symbol": "DAI",
+      "decimals": 18,
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/DAI.png?alt=media"
+    },
+    {
+      "tokenContract": "0x7af963cF6D228E564e2A0aA0DdBF06210B38615D",
+      "symbol": "TEST",
+      "decimals": 18,
+      "image": "https://firebasestorage.googleapis.com/v0/b/smart-invoice-b0db1.appspot.com/o/TEST.png?alt=media"
```

Comment:
Did you deploy a new TEST token to goerli? Can we get any $TEST to play with?

Reply:
this was an existing test token on goerli but looking closer not sure how reliable it is. I deployed our own test token (ITT) so I can send however much you want for testing.